tests/integration/Order/OrderUpdateRequestTest.php

```diff
@@ -852,488 +836,252 @@ function (Order $order) use ($client) {
+                $request = RequestBuilder::of()->orders()->update($order)
+                    ->addAction(
+                        OrderAddDeliveryAction::of()->setParcels(
+                            ParcelCollection::of()->add(
+                                Parcel::of()->setItems(
+                                    DeliveryItemCollection::of()->add(
+                                        DeliveryItem::of()
+                                            ->setId($lineItem->getId())
+                                            ->setQuantity(2)
+                                    )
+                                )
                             )
                         )
-                    )
-                )
-            )
-        ;
-        $response = $request->executeWithClient($this->getClient());
-        $order = $request->mapFromResponse($response);
-        $this->deleteRequest->setVersion($order->getVersion());
-
-        $parcel = $order->getShippingInfo()->getDeliveries()->current()->getParcels()->current();
-        $request = OrderUpdateRequest::ofIdAndVersion($order->getId(), $order->getVersion())
-            ->addAction(
-                OrderRemoveParcelFromDeliveryAction::ofParcel($parcel->getId())
-            )
-        ;
-        $response = $request->executeWithClient($this->getClient());
-        $order = $request->mapFromResponse($response);
-        $this->deleteRequest->setVersion($order->getVersion());
+                    );
+                $response = $this->execute($client, $request);
+                $order = $request->mapFromResponse($response);
 
-        $this->assertCount(
-            0,
-            $order->getShippingInfo()->getDeliveries()->current()->getParcels()
-        );
+                $parcel = $order->getShippingInfo()->getDeliveries()->current()->getParcels()->current();
 
+                $request = RequestBuilder::of()->orders()->update($order)
+                    ->addAction(
+                        OrderRemoveParcelFromDeliveryAction::ofParcel($parcel->getId())
+                    );
+                $response = $this->execute($client, $request);
+                $result = $request->mapFromResponse($response);
 
-        $request = ProductUpdateRequest::ofIdAndVersion($product2->getId(), $product2->getVersion())
-            ->addAction(ProductUnpublishAction::of());
-        $response = $request->executeWithClient($this->getClient());
-        $product2 = $request->mapResponse($response);
+                $this->assertCount(
+                    0,
+                    $result->getShippingInfo()->getDeliveries()->current()->getParcels()
+                );
 
-        $request = ProductDeleteRequest::ofIdAndVersion(
-            $product2->getId(),
-            $product2->getVersion()
+                return $result;
+            }
         );
-        $request->executeWithClient($this->getClient());
     }
 
     public function testCreateReplicaCartFromOrder()
```

Comment:
in cart request builder there is no method that takes the orderId to replicate the Cart